vendor/vendor.json

```diff
@@ -1935,6 +1941,10 @@
 			"path": "gopkg.in/yaml.v2",
 			"revision": "bef53efd0c76e49e6de55ead051f886bea7e9420"
 		},
+		{
```

Comment:
nit: This can be removed

Reply:
Hi @lebauce when removing it I get the below, but regardless, I would be happy to address (if needed) in a separate PR, as I need the k8s in the community version to support our internal integration efforts ....

```
make[1]: Entering directory '/home/vagrant/projects/gopath/src/github.com/skydive-project/skydive/contrib/snort'
govendor build snortSkydive.go
../../vendor/github.com/spf13/viper/util.go:29:2: cannot find package "gopkg.in/yaml.v2" in any of:
        /home/vagrant/projects/gopath/src/github.com/skydive-project/skydive/vendor/gopkg.in/yaml.v2 (vendor tree)
        /usr/local/go/src/gopkg.in/yaml.v2 (from $GOROOT)
        /home/vagrant/projects/gopath/src/gopkg.in/yaml.v2 (from $GOPATH)
Error: exit status 1
Makefile:6: recipe for target 'snortSkydive' failed
```